demo/site/src/layout/header/DesktopMenu.fragment.ts

```diff
@@ -0,0 +1,23 @@
+import { gql } from "@comet/site-nextjs";
+
+import { pageLinkFragment } from "./PageLink.fragment";
+
+export const desktopMenuFragment = gql`
+    fragment DesktopMenu on MainMenu {
+        items {
+            id
+            node {
+                id
+                name
+                ...PageLink
+                childNodes {
+                    id
+                    name
+                    ...PageLink
+                }
+            }
+        }
+    }
+
+    ${pageLinkFragment}
```

Comment:
I just tested this and currently, the app doesn't start because the `pageLinkFragment` is injected twice. Apparently, graphql can't deduplicate it even when they are identical. 

I suppose we can only fix this by removing pageLinkFragment from desktopMenuFragment and mobileMenuFragment and adding it to headerFragment instead